Spec/Auth.swift

```diff
@@ -1903,6 +1903,32 @@ class Auth : QuickSpec {
                         done()
                     }
                 }
+                
```

Comment:
I appreciate the code above was not yours, but I do not follow the logic.  The `queryTime` option is set to true for AuthOptions, I get that.  But the time from the `time` method is stored and then the authorise request is then executed and the issued attribute is checked to be within 1 second.  Firstly, if we have even a minor delay here this test will fail, but I am not sure what it's even testing.  If we want to check the time then we should have a test for http://docs.ably.io/client-lib-development-guide/features/#RSA10k, but this is mixing concerns and is brittle.  Please can you confirm why you think this is even here, and whether RSA10k has suitable coverage proving the request for time is made elsewhere i.e. we can then remove this.